crdppf/views/property.py

```diff
@@ -18,6 +18,9 @@ def get_property(request):
 
     proto = Protocol(DBSession, Property, 'geom')
 
-    filter = "id IN ('" + id_ + "')"
+    id_ = DBSession.query(Property.noobj).filter(Property.id==id_).first()
 
-    return proto.read(request, filter=filter)
+    if 'id_' == 0:

```

Comment:
That won't work...

`id_` is an object (or None if nothing is returned from the DB), not a string,...

```
if _id is None:
```
should work...